xc7/boards.cmake

```diff
@@ -1,19 +1,18 @@
 get_target_property_required(OPENOCD env OPENOCD)
 get_target_property_required(OPENOCD_TARGET env OPENOCD_TARGET)
 
-#define_board(
-#  BOARD basys3
-#  DEVICE xc7a50t
-#  PACKAGE test
-#  PROG_TOOL ${OPENOCD_TARGET}
-#  PROG_CMD "${OPENOCD} -f ${PRJXRAY_DIR}/utils/openocd/board-digilent-basys3.cfg -c \\\"init $<SEMICOLON> pld load 0 \${OUT_BIN} $<SEMICOLON> exit\\\""
-#)
-
+define_board(
+  BOARD basys3
+  DEVICE xc7a50t
+  PACKAGE test
+  PROG_TOOL ${OPENOCD_TARGET}
+  PROG_CMD "${OPENOCD} -f ${PRJXRAY_DIR}/utils/openocd/board-digilent-basys3.cfg -c \\\"init $<SEMICOLON> pld load 0 \${OUT_BIN} $<SEMICOLON> exit\\\""
+)
 
 define_board(
   BOARD zybo
   DEVICE xc7z010
   PACKAGE test
   PROG_TOOL ${OPENOCD_TARGET}
-  #  PROG_CMD "${OPENOCD} -f ${PRJXRAY_DIR}/utils/openocd/board-digilent-basys3.cfg -c \\\"init $<SEMICOLON> pld load 0 \${OUT_BIN} $<SEMICOLON> exit\\\""
+  PROG_CMD "${OPENOCD} -f ${PRJXRAY_DIR}/utils/openocd/board-digilent-basys3.cfg -c \\\"init $<SEMICOLON> pld load 0 \${OUT_BIN} $<SEMICOLON> exit\\\""
```

Comment:
Does this config work for the zybo?  If so, maybe rename it in PRJXRAY to board-digilent.cfg?

Reply:
I still need to verify if it works, but I doubt it

Reply:
Did this get resolved?

Reply:
No, but I would say that for now we could cut out the `PROG_TOOL` and `PROG_CMD` for now when taking into account the zybo. When there is something working we could think of this, I guess.

Reply:
That's fine, remove both lines to just have the prog tool be disabled for now.

Reply:
Also add a TODO comment and point to a new issue with symbiflow-arch-defs to get openocd working with zync.

Reply:
Done